build.py

```diff
@@ -77,7 +77,7 @@
     "ort_version": "1.20.1",
     "ort_openvino_version": "2024.4.0",
     "standalone_openvino_version": "2024.4.0",
-    "dcgm_version": "3.3.6",
+    "dcgm_version": "3.3.9",
```

Comment:
Do we need to update version for Model Analyzer? 
https://gitlab-master.nvidia.com/dl/triton/model_analyzer/-/blob/main/Dockerfile?ref_type=heads#L29

Reply:
It's need to be reverified. 
I see there is a condition, and probably it was working fine before some changes. 
But the catch itself worth attention.

Reply:
We should update MA to stay in sync, but doesn't need to block this PR. Will open a separate one for MA